src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Tables/Parameters.sql

```diff
@@ -0,0 +1,65 @@
+﻿-- Generic table to store application specific parameters
+-- DROP TABLE Parameters
+GO
+CREATE TABLE dbo.Parameters 
+  (
+     Id          varchar(100)     NOT NULL
+    ,Date        datetime         NULL
+    ,Number      float            NULL
+    ,Bigint      bigint           NULL
+    ,Char        varchar(4000)    NULL
+    ,Binary      varbinary(max)   NULL
+    
+    ,UpdatedDate datetime         NULL
+    ,UpdatedBy   nvarchar(255)    NULL
+    
+     CONSTRAINT PKC_Parameters_Id PRIMARY KEY CLUSTERED (Id)
+  )
+GO
+CREATE TABLE dbo.ParametersHistory 
```

Comment:
we might have to create a separate sql for this table(ParametersHistory.sql)

Reply:
I will do it only if snapshot sql file is generated incorrectly. I don't see it happening yet.